packages/config/src/Config.ts

```diff
@@ -405,6 +405,15 @@ export class Config {
         return this.layerMerge(true);
     }
 
+    /**
+     * Helper function to get masked properties and exclude the global layer
+     * @param excludeGlobal Indicates whether or not to exclude the global layer
+     * @returns Same as this.maskedProperties
+     */
+    public getMaskedProperties(excludeGlobal?: boolean): IConfig {
+        return this.layerMerge(true, excludeGlobal);
+    }
```

Comment:
Instead of adding this method, we may be able to make `layerMerge` a public internal method as we discussed 🙂

Reply:
done! 😋